stellar-contract-env-host/src/budget.rs

```diff
@@ -15,14 +15,15 @@ pub enum CostType {
 
 // TODO: add XDR support for iterating over all the elements of an enum
 impl CostType {
-    fn variants() -> std::slice::Iter<'static, CostType> {
+    pub fn variants() -> std::slice::Iter<'static, CostType> {
         static VARIANTS: &'static [CostType] = &[
             CostType::HostMapAllocMap,
             CostType::HostMapAllocCell,
             CostType::HostVecAllocVec,
             CostType::HostVecAllocCell,
             CostType::WasmInsnExec,
             CostType::WasmMemAlloc,
+            CostType::HostEventDebug,
```

Comment:
It's so odd that this part of the diff is showing up in the PR. GitHub must have a cache issue or a bug. The diff doesn't show this when viewed in git.